misc/openapi-ballerina/modules/openapi-to-ballerina-generator/src/test/java/org/ballerinalang/openapi/cmd/OpenApiGenServiceCmdTest.java

```diff
@@ -275,4 +275,101 @@ public void testInlineRequestBodyServiceGen() throws IOException {
         }
     }
 
+    @Test(description = "Test open-api genservice for successful service generation with all of schema type")
```

Comment:
It seems we don't have a license header in this file too. Check all the files in the package.

Reply:
Fixed with https://github.com/ballerina-platform/ballerina-lang/pull/20891